### PR TITLE
feat(manager): Adding total deployment count and avg time to deploy in the Application dashboard

### PR DESCRIPTION
**Pull request by nikhitamore**
## Explain the feature/fix

1. Adding total deployment count and avg time to deploy tiles at the top in the Application dashboard
2. making it non clickable

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

image

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Review by soumyadip007 — APPROVED** [review]
LGTM

File: packages/manager/src/services/analytics/queries.tsx [hunk 1]
```diff
@@ -209,76 +209,98 @@ export const useGetTotalDeployments = (propertyIdentifier?: string) =>
 const fetchTotalMonthlyDeploymentTime = async (
   propertyIdentifier?: string
 ): Promise<TSPADeploymentTime> => {
-  const { data } = await orchestratorReq.get('/analytics/deployment/time', {
-    params: {
-      propertyIdentifier,
-      days: '30'
-    }
-  });
+  const endpoint = propertyIdentifier
```
Comment (akhilmhdh) [inline packages/manager/src/services/analytics/queries.tsx:212]: use params for this in get options

File: packages/manager/src/services/analytics/queries.tsx [hunk 2]
```diff
@@ -209,76 +209,98 @@ export const useGetTotalDeployments = (propertyIdentifier?: string) =>
...
 const fetchTotalQuarterlyDeploymentTime = async (
   propertyIdentifier?: string
 ): Promise<TSPADeploymentTime> => {
-  const { data } = await orchestratorReq.get('analytics/deployment/time', {
-    params: {
-      propertyIdentifier,
-      days: '120'
-    }
-  });
+  const endpoint = propertyIdentifier
```
Comment (akhilmhdh) [inline packages/manager/src/services/analytics/queries.tsx:237]: Same

File: packages/manager/src/views/SPAPropertyDetailPage/SPAPropertyDetailPage.tsx [hunk 3]
```diff
@@ -84,7 +91,20 @@ export const SPAPropertyDetailPage = (): JSX.Element => {
...
   const lineChartLegend = Object.keys(monthlyDeployChart?.data || {}).map((key) => ({ name: key }));
+  const Totaldeployment = deploymentCount.data?.reduce((acc, obj) => acc + obj.count, 0);
 
+  const averageDeploymentTime = [
+    useGetMonthlyDeploymentsTime(propertyIdentifier, spaProperty).data,
+    useGetQuarterlyDeploymentsTime(propertyIdentifier, spaProperty).data,
+    useGetHalfYearlyDeploymentsTime(propertyIdentifier, spaProperty).data,
+    useGetYearlyDeploymentsTime(propertyIdentifier, spaProperty).data
+  ];
+  console.log('>>>', useGetMonthlyDeploymentsTime(propertyIdentifier, spaProperty).data);
```
Comment (akhilmhdh) [inline packages/manager/src/views/SPAPropertyDetailPage/SPAPropertyDetailPage.tsx:102]: Kindly remove the console log and the hook inside it

**Review by akhilmhdh — CHANGES_REQUESTED** [review]

File: packages/manager/src/views/SPAPropertyDetailPage/SPAPropertyDetailPage.tsx (hunk 3 above)
Comment (nikhitamore) [reply packages/manager/src/views/SPAPropertyDetailPage/SPAPropertyDetailPage.tsx]: done

**Review by akhilmhdh — APPROVED** [review]

**Pull request merged**